Operational-Procedures/Troubleshooting/component-offline.rst

```diff
@@ -0,0 +1,148 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Michael Reuter, Patrick Ingraham*
+
+.. _troubleshooting-component-offline:
+
+#################
+Component Offline
+#################
+
+.. warning::
+
+    This procedure contains steps that may affect the system in crical ways.
+
+    If you are unsure about any of the steps described here, contact support personnel.
+
+
+* Most (but not all) components in the Rubin Observatory Control System, when sent to OFFLINE will terminate its execution process and will no longer be available in the system.
+
+* This condition only happens if someone sends the ``exitControl`` command to the CSC.
+
+  Sending the ``exitControl`` command to a CSC is regularly done when one wants to update a version of the CSC or needs to restart the process to recover from some fault conditions.
+
+* A reduced number of CSCs will continue to run after receiving ``exitControl``.
+
+  At the time of this writting, only the camera systems have this behavior.
+
+Restoring a CSC that exited after responding to ``exitControl`` requires one to restart the process that runs it.
+
+This condition manifests by the CSC appearing in OFFLINE state with no heartbeats, and can be seen in the figure :ref:`below <fig-love-mtptg-offline>`.
+
+.. note::
+
+    A CSC may stop sending heartbeats for a number of different reasons while still executing (not in OFFLINE).
+
+    These cases are different from the one described here and may require different approaches.
+
+    If in doubt contact support personnel.
+
+.. figure:: ./_static/love-mtptg-offline.png
+    :name: fig-love-mtptg-offline
+
+    LOVE Summary State view showing the MTPtg in OFFLINE state with no heartbeats, indicating the CSC is no longer running.
+
+For more details about what is behind this behavior see the `observatory control`_ documentation page.
+
+.. _observatory control: https://obs-controls.lsst.io
+
+Most components on Rubin system runs as containers on a Kubernetes cluster.
+Therefore, most-likely, when restoring a component after it was sent to OFFLINE state, requires restarting the process on the Kubernetes cluster.
```

Comment:
Change to: Therefore, it is likely that restoring a component after it was sent to OFFLINE state requires restarting the process on the Kubernetes cluster.